main.c

```diff
@@ -149,11 +171,19 @@ int main(int argc, char **argv) {
         }
         bkinsert(bks[b], x, y);
         bka[b] = 0; /* BLOCK B NOT AVAILABLE */
-        for (c=0; c<3; c++) { /* CHOOSE NEW B */
-            if (b == c /* DON'T CHECK BKA[B] */
-             || !bka[c]) continue;
-            b = c;
-            break;
+        /* NO DEFAULT SELECTION UNLESS ONLY ONE BLOCK LEFT */
+        b = 0;
+        for (c=0; c<3; c++) { /* COUNT NUMBER OF AVAILABLE BLOCKS*/
+            b += bka[c];
+        }
```

Comment:
ooookay i just understood why you're doing this (selecting directly when there's only one left).
I feel like it makes sense but at the same time it slightly breaks the flow of choosing your block.

Testing if any block is left could be reduced to `if (bka[0] || bka[1] || bka[2])`
if you wanted the code to look homogeneously obfuscated lol (actually that feels more readable than temporarily using `b` to count available blocks).